docs/sources/awscodecommit.md

```diff
@@ -1,119 +1,123 @@
-variables under `env` with the values corresponding to your _AWS CodeCommit_ repository. Then, create that
-`ContainerSource` object in your Kubernetes cluster:
+```
+arn:aws:codecommit:{awsRegion}:{awsAccountId}:{repositoryName}
+```
 
-```console
-$ kubectl -n <my_namespace> create -f my-awscodecommit-containersource.yaml
+### API credentials
+
+The TriggerMesh AWS CodeCommit event source authenticates calls to the AWS CodeCommit API using an [Access
+Key][accesskey]. The page at this link contains instructions to create an access key when signed either as the root user
+or as an IAM user. Take note of the **Access Key ID** and **Secret Access Key**, they will be used to create and
+instance of the event source.
+
+It is considered a [good practice][iam-bestpractices] to create dedicated users with restricted privileges in order to
+programmatically access AWS services. Permissions can be added or revoked granularly for a given IAM user by attaching
+[IAM Policies][iam-policies] to it.
+
+As an example, the following policy contains only the permissions required by the TriggerMesh AWS CodeCommit event
+source to operate:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "codecommit:GetBranch",
+                "codecommit:GetCommit",
+                "codecommit:ListPullRequests",
+                "codecommit:GetPullRequest"
+            ],
+            "Resource": "arn:aws:codecommit:*:*:*"
+        }
+    ]
+}
 ```
 
-### As a Deployment object bound by a SinkBinding
+![Creating an IAM user](../images/awscodecommit-source/iam-user-1.png)
 
-Copy the sample manifest from `config/samples/awscodecommit-sinkbinding.yaml` and replace the pre-filled environment
-variables under `env` with the values corresponding to your _AWS CodeCommit_ repository. Then, create the
-`Deployment` and `SinkBinding` objects in your Kubernetes cluster:
+## Deploying an instance of the Source
 
-```console
-$ kubectl -n <my_namespace> create -f my-awscodecommit-sinkbinding.yaml
-```
+Open the Bridge creation screen and add a source of type `AWS CodeCommit`.
 
-## Running locally
+![Adding an AWS CodeCommit source](../images/awscodecommit-source/create-bridge-1.png)
 
-Running the event source on your local machine can be convenient for development purposes.
+In the Source creation form, give a name to the event source and add the following information:
 
-### In the shell
+* [**AWS ARN**][arn]: ARN of the CodeCommit repository, as described in the previous sections.
+* [**Branch name**][cc-branches]: Name of the Git branch the source should be watching for commits.
+* **Event types**: List of event types the event source should subscribe to.
+* [**AWS Secret**][accesskey]: Reference to a secret containing an Access Key ID and a Secret Access Key to communicate
```

Comment:
See here, the link goes to an AWS docs page but does not say how to create the secret on the triggermesh side

Reply:
Done in [bf97c766](https://github.com/triggermesh/docs/compare/80bed3913e9781b28750273e0892c2e40bdeb67e..bf97c7662a25053044023d85010f2ccfcad323a1)